cdc/cdc.hh

```diff
@@ -166,6 +166,21 @@ struct db_context final {
 /// param[in] schema schema of a table for which CDC tables are being created
 seastar::future<> setup(db_context ctx, schema_ptr schema);
 
+// cdc log table operation
+enum class operation : int8_t {
+    // note: these values will eventually be read by a third party, probably not privvy to this
+    // enum decl, so don't change the constant values. 
+    pre_image = 0, update = 1, row_delete = 2, range_delete_start = 3, range_delete_end = 4, partition_delete = 5
+};
+
+// cdc log data column operation
+enum class column_op : int8_t {
+    // same as above
+    set = 0, del = 1, add = 2,
+};
+
+seastar::future<> setup(service::storage_proxy& proxy, schema_ptr schema);
```

Comment:
Are you sure this is correctly rebased? on master this method is already present in cdc.hh -> https://github.com/scylladb/scylla/blob/master/cdc/cdc.hh#L167

Reply:
It uses db_context on master though.

Reply:
Of course not. Rebasing force-pushed stuff is a nightmare. I'll re-check.